sfcsample/VDecAccelVA.h

```diff
@@ -0,0 +1,154 @@
+/*
+ * * Copyright (C) 2018 Intel Corporation. All Rights Reserved.
+ * *
+ ** Permission is hereby granted, free of charge, to any person obtaining a
+ * * copy of this software and associated documentation files (the
+ * * "Software"), to deal in the Software without restriction, including
+ * * without limitation the rights to use, copy, modify, merge, publish,
+ * * distribute, sub license, and/or sell copies of the Software, and to
+ * * permit persons to whom the Software is furnished to do so, subject to
+ * * the following conditions:
+ * *
+ * * The above copyright notice and this permission notice (including the
+ * * next paragraph) shall be included in all copies or substantial portions
+ * * of the Software.
+ * *
+ * * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
+ * * IN NO EVENT SHALL PRECISION INSIGHT AND/OR ITS SUPPLIERS BE LIABLE FOR
+ * * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ * */
+
+/**
+ * @file VDecAccelVA.h
+ * @brief LibVA decode accelerator declaration
+ */
+
+#ifndef MV_ACCELERATOR_VAAPI_DECODE_H
+#define MV_ACCELERATOR_VAAPI_DECODE_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <map>
+#include <mutex>
+#include <vector>
+#include <algorithm>
+#include <va/va.h>
+#include "DecodeParamBuffer.h"
+
+#include <X11/Xlib.h>
+typedef Display*     HDISP;
+
+
+#define MVFOURCC(ch0, ch1, ch2, ch3)    (   \
```

Comment:
There is a similar macro in va/va.h VA_FOURCC

Reply:
remove MVFOURCC,  replace with VA_FOURCC in VDecAccelVA.cpp